src/main/resources/application-default.yml

```diff
@@ -0,0 +1,3 @@
+rebazer:
```

Comment:
This file should be packaged inside the debian package as well. Essentially copy the
`{basedir}/src/main/resources/application.yml`
element and adjust the filename.